### PR TITLE
Create conda environment

### PR DESCRIPTION
**Pull request by cazapatamar**
## 🛠️ Changes
- Create environment.yml and delete requirements.txt in order to use conda to manage dependencies
- Update Readme

## 📝 Associated issues
#13

### CODE REVIEW

**Comment by cazapatamar** [comment]
Tested in Kubuntu and Windows 10

**Review by erikasv — COMMENTED** [review]
Tested on ubuntu 22.04 and mac ventura ✅

Please @ManuelStardust or @denarch22 test and approve in case there are no issues

File: README.md [hunk 1]
```diff
@@ -28,9 +40,9 @@ Para usar correctamente la herramienta se deben preparar los insumos, a continua
 
 Para cargar una nueva colección (incluyendo los items de la misma) se debe seguir los siguientes pasos:
```
Comment (ManuelStardust) [inline README.md]: typo "deben"

File: README.md [hunk 2]
```diff
@@ -28,9 +40,9 @@ Para usar correctamente la herramienta se deben preparar los insumos, a continua
...
 
-1.  Lo primero que hay que hacer es describir toda la información que se desea cargar a la nueva colección, esto se hace por medio de un archivo `.json`, siguiendo la especificación descrita en el archivo [collection.md](spec/collection.md).
-
-    El archivo [collection.example.json](spec/collection.example.json) sirve como ejemplo y como punto de partida.
+1. Crear una carpeta, sin importar su nombre o ubicación, donde se va a almacenar todos los archivos necesarios para cargar la colección al STAC.
```
Comment (ManuelStardust) [inline README.md]: typo "van"

**Review by ManuelStardust — CHANGES_REQUESTED** [review]

**Comment by ManuelStardust** [comment]
Tested in Debian 12 ✅

**Review by ManuelStardust — APPROVED** [review]
Everything works fine

**Pull request merged**